index.md

```diff
@@ -7,86 +7,51 @@ title: MDAnalysis
+ -->
+**MDAnalysis** is an open-source Python library for analyzing molecular dynamics (MD) trajectories across [multiple formats]({{site.pypi.docs}}/documentation_pages/coordinates/init.html#id1). It enables seamless reading and writing of simulation data, allowing users to efficiently analyze molecular structures and dynamics, including particle-based trajectories and individual coordinate frames (e.g., biomolecules in the [PDB format][]). 
+
+With MDAnalysis, you can access atomic coordinates as [NumPy][] arrays, providing a flexible and efficient framework for complex analysis tasks. The library includes powerful [atom selection commands]({{site.pypi.docs}}/documentation_pages/selections.html) for extracting subsets of structures, supports trajectory transformations (e.g., fitting to a reference structure), and can write out trajectories along with [atom selections]({{site.pypi.docs}}/documentation_pages/selections_modules.html#selection-exporters) in formats suitable for visualization or native analysis tools.
+
+Learn more about our **mission, development, team, and governance** on the [About MDAnalysis]({{ site.baseurl }}about/) page.
+
+## Get Started  
+
+If you're new to MDAnalysis, explore the following resources to install the library, access tutorials, and dive into the documentation:
+
+- [Getting Started]({{ site.baseurl }}/pages/getting_started/)  
+- [Learning MDAnalysis]({{ site.baseurl }}/pages/learning_MDAnalysis)  
+- [Documentation]({{ site.baseurl }}/pages/documentation)  
+
+## Community Hub
+
+MDAnalysis is driven by an active **community of users and contributors**. Stay updated and get involved through the following pages:  
+
+<!-- TODO: Add link [Get Involved]({{ site.baseurl }}/pages/about/#get-involved)
+ -->
+- [Community]({{ site.baseurl }}/pages/community/) &mdash; Get involved, ask questions, and collaborate.
+- [Blog]({{ site.baseurl }}/blog) &mdash; Stay updated with development news and community highlights.
+- [Events]({{ site.baseurl }}/pages/events/) &mdash; Join workshops, conferences, and mentoring programs.
+
+## Contributing 
+
+Want to contribute to MDAnalysis? Here’s how:  
+
+- [Contribute]({{ site.baseurl }}/pages/contribute/) &mdash; Learn how to help improve MDAnalysis through coding, documentation, or discussions.  
+- [MDAKits and MDA-based tools]({{ site.baseurl }}/pages/mdakits/) &mdash; Build and extend MDAnalysis with MDAKits.  
+
+## Citing MDAnalysis
+
+If you use MDAnalysis in your research, please cite it appropriately and consider displaying our MDAnalysis badge in your projects. All citation formats and badge instructions are available in our [Citation guidelines]({{ site.baseurl }}/pages/citations/).
+
+## Funding & Support
+
+MDAnalysis has received generous support from organizations such as the **Chan Zuckerberg Initiative, Google, NumFOCUS, and the National Science Foundation**.  
```

Comment:
I would mention 'numfocus sponsored project' before support from any other grants

Reply:
I think it should be in the 1st paragraph of the page.